Test test_rwx_pvc_assign_pod_node failing in ODF 4.10 on MS

tests.manage.pv_services.test_pvc_assign_pod_node.TestPvcAssignPodNode.test_rwx_pvc_assign_pod_node
```
>       assert not (
            error_msg in pod_log
        ), f"Logs should not contain the error message '{error_msg}'"
E       AssertionError: Logs should not contain the error message 'Authorization: Bearer'
E       assert not 'Authorization: Bearer' in 'I0213 11:02:34.183823       1 main.go:181] Valid token audiences: \nI0213 11:02:34.184099       1 main.go:289] Genera...-4038-92c3-6d1ac09669fd"]}},"audiences":["https://d3gt1gce2zmg3d.cloudfront.net/21rq6tah1ocs3fmuocbtres2gmq4ga9a"]}}\n'

tests/manage/pv_services/test_pvc_assign_pod_node.py:46: AssertionError
```

--- a/tests/manage/pv_services/test_pvc_assign_pod_node.py
+++ b/tests/manage/pv_services/test_pvc_assign_pod_node.py
@@ -8,13 +8,13 @@ from ocs_ci.framework.testlib import (
     tier1,
     acceptance,
     bugzilla,
-    skipif_ocs_version,
 )
 from ocs_ci.ocs import constants
 from ocs_ci.ocs.node import get_worker_nodes
 from ocs_ci.ocs.resources import pod
 from ocs_ci.helpers import helpers
 from ocs_ci.ocs.resources import pod as res_pod
+from ocs_ci.utility import version
 
 logger = logging.getLogger(__name__)
 
@@ -28,7 +28,6 @@ class TestPvcAssignPodNode(ManageTest):
     OCS-1257 - RBD: Assign nodeName to a POD using RWX PVC
     """
 
-    @skipif_ocs_version("<4.12")
     def verify_access_token_notin_odf_pod_logs(self):
         """
         This function will verify logs of kube-rbac-proxy container in odf-operator-controller-manager pod
@@ -101,7 +100,9 @@ class TestPvcAssignPodNode(ManageTest):
         pod_obj.run_io(storage_type="fs", size="512M", runtime=30, invalidate=0)
         pod.get_fio_rw_iops(pod_obj)
 
-        self.verify_access_token_notin_odf_pod_logs()
+        ocs_version = version.get_semantic_ocs_version_from_config()
+        if ocs_version >= version.VERSION_4_12:
+            self.verify_access_token_notin_odf_pod_logs()
 
     @acceptance
     @tier1
@@ -188,4 +189,6 @@ class TestPvcAssignPodNode(ManageTest):
         for pod_obj in pod_list:
             pod.get_fio_rw_iops(pod_obj)
 
-        self.verify_access_token_notin_odf_pod_logs()
+        ocs_version = version.get_semantic_ocs_version_from_config()
+        if ocs_version >= version.VERSION_4_12:
+            self.verify_access_token_notin_odf_pod_logs()

